### PR TITLE
Bug/AddingExtraLoging, there is not enough logs to investigate timeou…

### PR DESCRIPTION
**Pull request by indu2019**
I've raised below ticket for em team, and they said that there is not enough login to investigate it further, So I am adding extra logs.
https://tools.hmcts.net/jira/browse/EM-4586

### Change description ###
Adding extra logs.... this change got two extra code lines.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 1]
```diff
@@ -51,17 +51,19 @@ public CaseDocument bundle(
                 bundleFilename,
                 caseData
             );
-
+        log.info("calling stitching api end point for " + caseData.getLegacyCaseReference());
```
Comment (mounikahmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:54]: logger should be in this format 
            log.info(String.format("Http Status %s ", e.status()), e); .. appending is not a good way of adding logs.

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 2]
```diff
@@ -51,17 +51,19 @@ public CaseDocument bundle(
...
         CaseData caseData1 =
             bundleRequestExecutor.post(
                 BundleRequest.builder().caseDetails(payload).build(),
                 stitchingConfiguration.getStitchingUrl(),
                 authorisation
             );
+        log.info("called stitching api end point for " + caseData.getLegacyCaseReference());
```
Comment (mounikahmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:61]: same here

**Review by mounikahmcts — CHANGES_REQUESTED** [review]

**Review by asthamalviya — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java (hunk 1 above)
Comment (indu2019) [reply src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:54]: made changes

**Review by mounikahmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 3]
```diff
@@ -51,17 +51,19 @@ public CaseDocument bundle(
                 bundleFilename,
                 caseData
             );
-
+        log.info("calling stitching api end point for ", caseData.getLegacyCaseReference());
```
Comment (mounikahmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:54]: @indu2019 this log doesnt work .. it should be 
        log.info("Received ActorId: {}", actorId); like this ..Did you check it locally

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 4]
```diff
@@ -51,17 +51,19 @@ public CaseDocument bundle(
...
         CaseData caseData1 =
             bundleRequestExecutor.post(
                 BundleRequest.builder().caseDetails(payload).build(),
                 stitchingConfiguration.getStitchingUrl(),
                 authorisation
             );
+        log.info("called stitching api end point for ", caseData.getLegacyCaseReference());
```
Comment (mounikahmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:61]: same here

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 5]
```diff
@@ -51,17 +51,19 @@ public CaseDocument bundle(
...
         if (caseData1 != null) {
             Optional<Document> stitchedDocument = caseData1.getCaseBundles().get(0).getValue().getStitchedDocument();
 
-            log.info("stitchedDocument.isPresent()----->" + stitchedDocument.isPresent());
+            log.info("stitchedDocument.isPresent()----->",  stitchedDocument.isPresent() + " for ",
```
Comment (mounikahmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:65]: this is again a different format

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java (hunk 3 above)
Comment (mounikahmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:54]: For more info : read here https://www.slf4j.org/faq.html#logging_performance

**Pull request merged**